admin/database/database.go

```diff
@@ -607,18 +607,20 @@ type InsertDeploymentAuthTokenOptions struct {
 
 // MagicAuthToken is a persistent API token for accessing a specific (filtered) resource in a project.
 type MagicAuthToken struct {
-	ID                    string
-	SecretHash            []byte         `db:"secret_hash"`
-	ProjectID             string         `db:"project_id"`
-	CreatedOn             time.Time      `db:"created_on"`
-	ExpiresOn             *time.Time     `db:"expires_on"`
-	UsedOn                time.Time      `db:"used_on"`
-	CreatedByUserID       *string        `db:"created_by_user_id"`
-	Attributes            map[string]any `db:"attributes"`
-	MetricsView           string         `db:"metrics_view"`
-	MetricsViewFilterJSON string         `db:"metrics_view_filter_json"`
-	MetricsViewFields     []string       `db:"metrics_view_fields"`
-	State                 string         `db:"state"`
+	ID                      string
+	SecretHash              []byte         `db:"secret_hash"`
+	TokenStr                string         `db:"token_str"`
+	TokenStrEncryptionKeyID string         `db:"token_str_encryption_key_id"`
```

Comment:
For consistency with `SecretHash`, I would suggest calling it `Secret` instead of `TokenStr`. Also, should it be a `[]byte` instead of a `string`?

The auth code generally uses "token" to mean a serialized access token in the form `rill_<type>_<id><secret>`, but in this case it's only the `secret` part that is being encrypted and persisted.

Reply:
Will change it to secret, however its a `string` because encrypt/decrypt method accepts string and return string. So thought one more conversion is unnecessary, however if we want to be consistent with `SecretHash` then can convert back to `[]byte` or change encrypt/decrypt to accept `[]byte` and return `[]byte` but then prod vars needs to be converted to []byte before encrypting, so probably should go with more frequent usage.

Reply:
Since encryption outputs bytes, it feels like it makes more sense just to store those.

The prod vars just apply base64 on top of the encrypted bytes, right? So you could just have `encryptAsString` and `decryptFromString` or something like that, which wraps the `[]byte` implementation with base64. Or maybe there are simpler approaches, just an example.